internal/kv/etcd/etcd_kv.go

```diff
@@ -140,51 +140,6 @@ func (kv *EtcdKV) LoadBytesWithPrefix(key string) ([]string, [][]byte, error) {
 	return keys, values, nil
 }
 
```

Comment:
We can rename EtcdKV to etcdKV so that other packages won't misuse it.

Reply:
Sounds reasonable. 
I did a quick code search. It looks like EtcdKV is used in many tests. Let's make the change in a separate PR as this one is already of size/XL.